plugins/packs-integrations.js

```diff
@@ -169,14 +169,17 @@ async function fetchPackListItems(queryParams, packDataArr, counter) {
 }
 
 async function mapRepositories(repositories) {
-  const results = await api.get('v1/registries/pack');
+  const urls = ["v1/registries/oci/summary", "v1/registries/pack"];
+  const results = await callRateLimitAPI(() => Promise.all(urls.map((url) => api.get(url))));
```

Comment:
this complicates things unnecessarily.
cont ociRegistries = await api.get("v1/registries/oci/summar")
const packRegistries = await api.get("v1/registries/pack")
const results = [...ociRegistries, ...packRegistries]

Reply:
you won't hit the api limit with this